# Three-phase Power Systems

### *Chapter 10 - Polyphase AC Circuits*

## What are Split-Phase Power Systems?

Split-phase power systems achieve their high conductor efficiency and low safety risk by splitting up the total voltage into lesser parts and powering multiple loads at those lesser voltages while drawing currents at levels typical of a full-voltage system.

This technique, by the way, works just as well for DC power systems as it does for single-phase AC systems. Such systems are usually referred to as three-wire systems rather than split-phase because “phase” is a concept restricted to AC.

But we know from our experience with vectors and complex numbers that AC voltages don’t always add up as we think they would if they are out of phase with each other.

This principle, applied to power systems, can be put to use to make power systems with even greater conductor efficiencies and lower shock hazard than with split-phase.

# Examples

## Two 120° Out of Phase Voltage Sources

Suppose that we had two sources of AC voltage connected in series just like the split-phase system we saw before, except that each voltage source was 120° out of phase with the other: (Figure below)

![example-1](https://www.allaboutcircuits.com/uploads/articles/phase-voltage-sources-example.jpg)

### Verification

Here we verify that the voltage drop from the positive pole of load \#1 to the positive pole of load \#2, henceforth '$E$', is indeed equal to $207.85\mathrm{V}\angle\,-\!30^\circ$.

$$ E = 120\mathrm{V}\angle\,0 - 120 \mathrm{V}\angle\,120^\circ = 120\mathrm{V}\angle\,0 + 120\mathrm{V}\angle\,-\!60^\circ$$

In rectangular form:
$$ E = 120\mathrm{V} + 120\mathrm{V}\big(\cos-60^\circ + j\cdot \sin-60^\circ \big)$$

In [1]:
from math import pi, cos, sin, radians, degrees

In [2]:
(cos(radians(-60)), sin(radians(-60)))

(0.5000000000000001, -0.8660254037844386)

Thus,
$$ E \approxeq 120\mathrm{V}\big( 1.5 - j\cdot0.866\big)$$

Finally, in polar form:
$$ E = |E|\angle\,E_\theta$$

where,

$$|E| = 120\mathrm{V}\,\sqrt{1.5^2 + 0.866^2} \approxeq 207.84 \mathrm{V}$$

and,
$$ E_\theta = \arcsin - \frac{0.866}{\sqrt{1.5^2+0.866^2}} \approxeq -30^\circ$$

In [3]:
import numpy as np

In [4]:
120*np.sqrt(1.5**2 + 0.866**2)

207.84457269796582

In [5]:
np.degrees(np.arctan(-0.866/1.5))

-29.999272229846984

### Findings and Conclusions

So, we find that the “neutral” wire is carrying a full 83.33 amps, just like each “hot” wire.

Note that we are still conveying 20 kW of total power to the two loads, with each load’s “hot” wire carrying 83.33 amps as before.

With the same amount of current through each “hot” wire, we must use the same gauge copper conductors, so we haven’t reduced system cost over the split-phase 120/240 system.

However, we have realized a gain in safety, because the overall voltage between the two “hot” conductors is 32 volts lower than it was in the split-phase system (208 volts instead of 240 volts).

## Three 120° Out of Phase Voltage Sources

The fact that the neutral wire is carrying 83.33 amps of current raises an interesting possibility: since it's carrying current anyway, why not use that third wire as another “hot” conductor, powering another load resistor with a third 120 volt source having a phase angle of 240°?

That way, we could transmit more power (another 10 kW) without having to add any more conductors. Let’s see how this might look: (Figure below)

![example-1.2](https://www.allaboutcircuits.com/uploads/articles/phase-voltage-sources-example1.jpg)

### Circuit Analysis

The only quantity that needs to be found for a complete analysis of the above circuit is the resistance of each load.  

$$ P = \frac{E^2}{R} = 10\mathrm{kW}$$

$$ R = \frac{E^2}{P} = \frac{120^2\mathrm{V}^2}{10\mathrm{kW}}$$

In [6]:
120**2/10000

1.44

$$ R = 1.44\Omega$$

Note that the above applies to each load.  

![analysed-example](https://www.allaboutcircuits.com/uploads/articles/spice-circuit1.jpg)

Now, it is important to note that there is no longer a "neutral" conductor, but we want one for safety reasons and for voltage stability;  If one load were to fail open, the voltage dropped across the rest of the loads would change accordingly.  To fix this we'll add a "neutral" wire to the above circuit.

![example-1.2-w/neutral](https://www.allaboutcircuits.com/uploads/articles/spice-circuit-annotated-with-simulation-results.jpg)

So long as the loads remain balanced (equal resistance, equal currents), the neutral wire will not have to carry any current at all. It is there just in case one or more load resistors should fail open (or be shut off through a disconnecting switch).

## Polyphase Circuit

This circuit we’ve been analyzing with three voltage sources is called a polyphase circuit. The prefix “poly” simply means “more than one,” as in “polytheism” (belief in more than one deity), “polygon” (a geometrical shape made of multiple line segments: for example, pentagon and hexagon), and “polyatomic” (a substance composed of multiple types of atoms).

Since the voltage sources are all at different phase angles (in this case, three different phase angles), this is a “polyphase” circuit.

More specifically, it is a three-phase circuit, the kind used predominantly in large power distribution systems.

## Three-Phase System versus Single-Phase System

#### Single-Phase System

Let’s survey the advantages of a three-phase power system over a single-phase system of equivalent load voltage and power capacity. A single-phase system with three loads connected directly in parallel would have a very high total current (83.33 times 3, or 250 amps. (Figure below)

![single-phase-example-diagram](https://www.allaboutcircuits.com/uploads/articles/single-phase-system-with-three-loads.jpg)

This would necessitate 3/0 gage copper wire (very large!), at about 510 pounds per thousand feet, and with a considerable price tag attached. If the distance from source to load was 1000 feet, we would need over a half-ton of copper wire to do the job.

#### Split-phase System

On the other hand, we could build a split-phase system with two 15 kW, 120 volt loads. (Figure below)

![split-phase-example-diagram](https://www.allaboutcircuits.com/uploads/articles/split-phase-system.jpg)